examples/gke/deploy-paligemma-2-with-tgi/README.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Deploy PaliGemma2 with TGI DLC on GKE
+type: inference
+---
+
+# Deploy PaliGemma2 with TGI DLC on GKE
+
+PaliGemma 2 is the latest multilingual vision-language model released by Google, that combines the Gemma2 language model with the SigLIP vision model, enabling it to process both images and text inputs to generate text outputs for various tasks including captioning, visual question answering, and object detection. Text Generation Inference (TGI) is a toolkit developed by Hugging Face for deploying and serving LLMs, with high performance text generation. Google Kubernetes Engine (GKE) is a fully-managed Kubernetes service in Google Cloud that can be used to deploy and operate containerized applications at scale using Google Cloud infrastructure.
```

Comment:
We should mention, that the pt model needs to be fine-tuned before use and also use DOCCI checkpoint here otherwise model outputs will not be as good
if you feel like it you can use my VQAv2 finetuned model too as it's fine-tuned with transformers

Reply:
Great, I'll add a note here (I believe it's also mentioned below too), and point to both the VQA and DOCCI fine-tunes too 👍🏻